### ISSUE TITLE
Merge with Old system

### ISSUE THREAD

**Issue by itsMagondu** [PeterNduati84/cs2.0#1]
Hi,

I think we should merge with the other repo to avoid having two code bases to maintain and also to ensure we can track the changes over time. What do you think? I believe github was made to ensure you only have one code base to maintain.

**Comment by PeterNduati84** [comment]
What l think will do is have the two running as branches ,then once we have finished making all the changes will version and merge into one main branch. Could not merge the two cause of the difference in code and also seeing the older one had been written in windows ,there could be a problem with handling files.

**Comment by itsMagondu** [comment]
I agree. Having them as branches is easier to manage. Even we can track issues at one place.

**Comment by PeterNduati84** [comment]
Will see to it that its done . Will also show you the progress on Sunday.

**Comment by itsMagondu** [comment]
Great. We should then go ahead and close this repository and have the one repository. Also is it ok if I work on a concept for registration with Django forms?
